code/modules/spacepods/spacepod.dm

```diff
@@ -73,7 +73,7 @@
 	pr_int_temp_processor = new /datum/global_iterator/pod_preserve_temp(list(src))
 	pr_give_air = new /datum/global_iterator/pod_tank_give_air(list(src))
 	equipment_system = new(src)
-	equipment_system.installed_modules = new/list()
+	equipment_system.installed_modules = list()
```

Comment:
@tigercat2000 
Are you sure about this? list() declares the variable is a list but doesn't create the list... you need either new/list() or list(0) to create the list as well, and as the next line tries to add something to the list it needs to be created